app/src/main/java/com/hadiyarajesh/notex/ui/note/NotesViewModel.kt

```diff
@@ -14,4 +23,41 @@ class NotesViewModel @Inject constructor(
     init {
         Log.i(TAG, "${this.javaClass.name} initialized")
     }
+
+    lateinit var notesFlow: StateFlow<List<Note>>
```

Comment:
Why `lateinit` here?
I'd already provided that implementation with paging data
```kotlin
 val notes: Flow<PagingData<Note>> =
        notesRepository
            .getAllNotes()
            .cachedIn(viewModelScope)
```

Even if you need to create `StateFlow` I've already provided `UiState` class which you can use to initialise state of ui

```kotlin
private val _notes: MutableStateFlow<UiState<List<Note>> = MutableStateFlow(UiState.Empty)
val notes: StateFlow<UiState<List<Note>>> = _notes.asStateFlow()
```